src/EncompassRest/EncompassRestClient.cs

```diff
@@ -97,6 +97,11 @@ public interface IEncompassRestClient : IDisposable
         /// </summary>
         CommonCache CommonCache { get; }
 
+        /// <summary>
+        /// Set by ClientParameters.BaseAddress.  The URL to call for API calls.  Defaults to "https://api.elliemae.com/".
+        /// </summary>
+        string BaseAddress { get; }
```

Comment:
Let's make this a getter and setter.

Reply:
Done.  This will allow for the changing of the BaseAddress "after" they have a client instance.